spec/models/movie_spec.rb

```diff
@@ -2,6 +2,8 @@
 
 RSpec.describe Movie, type: :model do
   describe "relationships" do
-    it {should belong_to :studio}
+    it { should belong_to :studio }
+    it { should have_many :movie_actors }
+    it { should have_many(:actors).through(:movie_actors)}
```

Comment:
These relationship tests aer good, but try to get into the habit of checking your simplecov coverage after each user story at the very least. I know you commented that you forgot the model tests, but you definitely want to remember them for the final! 
The command for checking the coverage at any time is: 
```bash
open coverage/index.html
```

Reply:
I also made an alias in my Terminal for myself to remember easier, called `cov` that runs that command for me. You may want to play with those aliases too. https://github.com/cjsim89/dotfiles